package.json

```diff
@@ -2,44 +2,57 @@
   "name": "@dashevo/dapi-client",
   "version": "0.13.0-dev.3",
   "description": "Client library used to access Dash DAPI endpoints",
-  "main": "index.js",
+  "main": "lib/DAPIClient.js",
+  "contributors": [
+    {
+      "name": "Ivan Shumkov",
+      "email": "ivan@shumkov.ru",
+      "url": "https://github.com/shumkov"
+    },
+    {
+      "name": "Djavid Gabibiyan",
+      "email": "djavid@dash.org",
+      "url": "https://github.com/jawid-h"
+    },
+    {
+      "name": "Anton Suprunchuk",
+      "email": "anton.suprunchuk@dash.org",
+      "url": "https://github.com/antouhou"
+    },
+    {
+      "name": "Konstantin Shuplenkov",
+      "email": "konstantin.shuplenkov@dash.org",
+      "url": "https://github.com/shuplenkov"
+    }
+  ],
   "dependencies": {
-    "@babel/polyfill": "^7.8.3",
```

Comment:
What exactly is the reason to remove babel runtime?

Reply:
It works without it :)

Reply:
Decided that we will proceed without polyfils for now, in case of any issues in the future we'll bring it back.